src/sb_pointcloud_processing/include/DBSCAN.h

```diff
@@ -7,6 +7,8 @@
 #ifndef LINE_EXTRACTOR_IGVC_DBSCAN_H
 #define LINE_EXTRACTOR_IGVC_DBSCAN_H
 
+#define SEQUENTIAL_CUT_OFF 5000
```

Comment:
What was the reasoning behind making this a pre-processor definition? If possible we should make this a settable class member

Reply:
A comment explaining what this is used for would also be really helpful.

Reply:
I made the sequential cut off a pre-processor definition because it's far more unlikely that we would be constantly changing this value unlike some of the other sensitive hyperparameters we have for DBSCAN like `radius` and `min_neighbours`. Also "standard" values for sequential cut off ranges from 1000 to 5000. That was my reasoning, let me know if you still think we should make this a configurable value

Reply:
Hmm, I see what you mean, but I don't think it'd hurt to make it a class member, at the very least it might be a bit more clear that this is part of the class itself.